v-next/hardhat-test-utils/src/hardhat-error.ts

```diff
@@ -44,18 +60,27 @@ export function assertThrowsHardhatError<
   assert.fail("Function did not throw any error");
 }
 
+/**
+ * Asserts that an async operation (i.e. calling an async function or a promise)
+ * gets rejected with a HardhatError with a certain descriptor and message
+ * arguments.
+ *
+ * @param op The async operation. If it's a function, it's called and awaited.
+ * @param descritor The error descriptor.
+ * @param messageArguments The message arguments.
+ */
 export async function assertRejectsWithHardhatError<
   ErrorDescriptorT extends ErrorDescriptor,
 >(
-  f: (() => Promise<any>) | Promise<any>,
+  op: (() => Promise<any>) | Promise<any>,
   descritor: ErrorDescriptorT,
```

Comment:
oops, we missed this in the previous PR. There's a typo here, and it's also present in the other function.